
From: Geert Uytterhoeven <geert@linux-m68k.org>

Atari ACSI: Correct a small problem in the dependencies of ATARI_BIONET and
ATARI_PAMSNET (e.g.  ATARI_ACSI=m shouldn't allow ATARI_BIONET=y).

Signed-off-by: Adrian Bunk <bunk@fs.tum.de>
Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 /dev/null                   |    0 
 25-akpm/drivers/net/Kconfig |    4 ++--
 2 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/net/Kconfig~atari-acsi-dependencies drivers/net/Kconfig
--- 25/drivers/net/Kconfig~atari-acsi-dependencies	2004-10-02 18:40:49.682372960 -0700
+++ 25-akpm/drivers/net/Kconfig	2004-10-02 18:40:49.687372200 -0700
@@ -391,7 +391,7 @@ config ATARILANCE
 
 config ATARI_BIONET
 	tristate "BioNet-100 support"
-	depends on NETDEVICES && ATARI && ATARI_ACSI!=n && BROKEN
+	depends on NETDEVICES && ATARI && ATARI_ACSI && BROKEN
 	help
 	  Say Y to include support for BioData's BioNet-100 Ethernet adapter
 	  for the ACSI port. The driver works (has to work...) with a polled
@@ -399,7 +399,7 @@ config ATARI_BIONET
 
 config ATARI_PAMSNET
 	tristate "PAMsNet support"
-	depends on NETDEVICES && ATARI && ATARI_ACSI!=n && BROKEN
+	depends on NETDEVICES && ATARI && ATARI_ACSI && BROKEN
 	help
 	  Say Y to include support for the PAMsNet Ethernet adapter for the
 	  ACSI port ("ACSI node"). The driver works (has to work...) with a
diff -L drivers/net/Kconfig.old -puN /dev/null /dev/null
_
